.github/workflows/ci-additional.yml

```
@@ -16,9 +16,11 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Setup Conda Environment
-        uses: conda-incubator/setup-miniconda@v2
+        uses: conda-incubator/setup-miniconda@master
```

If you want this now, you can pin this to a commit:

```suggestion
        uses: conda-incubator/setup-miniconda@77b16ed746da28724c61e1f1ad23395a4b695ef5
```

and then make a follow up PR when a new version gets tagged.